lib/motors/pid_controller.hpp

```diff
@@ -0,0 +1,44 @@
+#include "core/types.hpp"
+#pragma once
```

Comment:
`pragma` above the includes, add whitespace between `pragma`, `includes`, and `namespace`